cluster/userdata-master.yaml

```diff
@@ -363,13 +363,47 @@ write_files:
             - name: CLUSTER_ID
               value: {{WEBHOOK_ID}}
             - name: TOKENINFO_URL
-              value: https://info.services.auth.zalando.com/oauth2/tokeninfo
+              value: http://127.0.0.1:9021/oauth2/introspect
             - name: USER_GROUPS
               value: stups_autobahn=Administrator,stups_deployment-controller-ui=ReadOnly,credentials-provider=Administrator,credprov-kube-ops-view-read-only-token=ReadOnly,credprov-cluster-lifecycle-manager-cluster-rw-token=Administrator,credprov-cluster-lifecycle-manager-test-cluster-rw-token=Administrator,credprov-cdp-controller-cluster-token=PowerUser
           volumeMounts:
           - mountPath: /etc/kubernetes/config
             name: kubernetes-configs
             readOnly: true
+        - image: pierone.stups.zalan.do/foundation/platform-iam-tokeninfo:46fa3a9
```

Comment:
Can we get tokeninfo from the opensource registry?

Otherwise we have a chicken and egg problem because we can't pull from pierone before secretary is deployed, and we can't deploy secretary without tokeninfo (CLM needs access).